package.json

```diff
@@ -7,6 +7,7 @@
     "test": "test"
   },
   "devDependencies": {
+    "core-assert": "^0.1.3",
```

Comment:
Incorrect in the sense that you did not use `~` like the other dependencies do. When in doubt look at adjacent code/style and be consistent with it leaving whatever your other projects or preferences may be out.